README.md

```diff
@@ -121,16 +113,10 @@ for more information about how to generate these.
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware         => true,
-    middleware_hosts   => [ 'broker1.example.com' ],
-    middleware_ssl     => true,
-    securityprovider   => 'ssl',
-    ssl_client_certs   => 'puppet:///modules/site_mcollective/client_certs',
-    ssl_ca_cert        => 'puppet:///modules/site_mcollective/certs/ca.pem',
-    ssl_server_public  => 'puppet:///modules/site_mcollective/certs/server.pem',
-    ssl_server_private => 'puppet:///modules/site_mcollective/private_keys/server.pem',
-  }
+  # Please see
+  # https://github.com/puppetlabs/puppetlabs-mcollective/blob/master/examples/mco_profile/manifests/middleware/activemq.pp
+  # for this as the changes to template and neccessity to build a
+  # keystore makes this quite complex.
```

Comment:
Should this be in the regular markup text rather than a comment in a code example?